### PR TITLE
OFX Mapping fix to enable support for Money Pro

### PR DESCRIPTION
**Pull request by addomafi**
I've made some changes on the mapping of OFX file to be able to import transactions on application Money Pro for Mac OS X.

Another change regarding to the amount, where spends should be a negative value.

### CODE REVIEW

**Comment by lucasprag** [comment]
Nice! ping @gushonorato

File: extrato-nubank.js [hunk 1]
```diff
@@ -14,30 +14,50 @@ OLDFILEUID:NONE
...
+            <ACCTID>0</ACCTID>
+            <ACCTTYPE>LINEOFCREDIT</ACCTTYPE>
+        </BANKACCTFROM>
+        <BANKTRANLIST>
+          <DTSTART>${from}</DTSTART>
+          <DTEND>${to}</DTEND>`;
+          }
+        
+          function endOfx() {
+            return `
```
Comment (tonicospinelli) [inline extrato-nubank.js:44]: could you fix spaces here, pls?

File: extrato-nubank.js [hunk 2]
```diff
@@ -14,30 +14,50 @@ OLDFILEUID:NONE
...
-<STMTTRN>
-<TRNTYPE>OTHER</TRNTYPE>
-<DTPOSTED>${date}</DTPOSTED>
-<TRNAMT>${amount}</TRNAMT>
-<MEMO>${description}</MEMO>
-</STMTTRN>`;
+      <STMTTRN>
+      <TRNTYPE>OTHER</TRNTYPE>
+      <DTPOSTED>${date}</DTPOSTED>
+      <TRNAMT>${amount * -1}</TRNAMT>
```
Comment (tonicospinelli) [inline extrato-nubank.js:58]: 👍

**Review by tonicospinelli — CHANGES_REQUESTED** [review]
I'm not OFX specialist, but I think the lines that you added is only to have a "header", right?

**Comment by marlonanjos** [comment]
@addomafi I have forked this project to focus on compatibility with YNAB (where spends must be negative) and wanted to add Money Pro compatibility, are u still using it?

**Comment by gushonorato** [comment]
@marlonanjos nice! Thanks! I don't use Money Pro anymore, but I can accept the changes for other people that use it. Can you fix the conflicts?

**Comment by addomafi** [comment]
> @addomafi I have forked this project to focus on compatibility with YNAB (where spends must be negative) and wanted to add Money Pro compatibility, are u still using it?

Sorry i leave it... now I'm using the file provided from Nubank!

**Pull request closed without merging**